mesonbuild/modules/gnome.py

```diff
@@ -1138,7 +1138,10 @@ def generate_vapi(self, state, args, kwargs):
         build_dir = os.path.join(state.environment.get_build_dir(), state.subdir)
         source_dir = os.path.join(state.environment.get_source_dir(), state.subdir)
         pkg_cmd, vapi_depends, vapi_packages, vapi_includes = self._extract_vapi_packages(state, kwargs)
-        cmd = [find_program('vapigen', 'Vaapi')]
+        if 'VAPIGEN' in os.environ:
+            cmd = [find_program(os.environ['VAPIGEN'], 'Vaapi')]
+        else:
+            cmd = [find_program('vapigen', 'Vaapi')]
```

Comment:
Yes, you are right, this typo was made in the very beginning, with e933bdd87229f13766705274a8815e9f10d4fe8d. It should be `'generate_vapi({})'.format(library)`.